public/app/features/alerting/unified/components/rule-viewer/Actions.tsx

```diff
@@ -43,46 +60,49 @@ export const useAlertRulePageActions = ({ handleDelete, handleDuplicateRule }: P
    * Since Incident isn't available as an open-source product we shouldn't show it for Open-Source licenced editions of Grafana.
    * We should show it in development mode
    */
-  const shouldShowDeclareIncidentButton = !isOpenSourceEdition() || isLocalDevEnv();
+  const shouldShowDeclareIncidentButton =
+    (!isOpenSourceEdition() || isLocalDevEnv()) &&
+    isAlertingRule(rule.promRule) &&
+    rule.promRule.state === PromAlertingRuleState.Firing;
```

Comment:
Bringing in [the previous logic](https://github.com/grafana/grafana/pull/86071/files#diff-691394a56baf1c8cb3534abb9530db44e1329d6f0bd8866049404848779bd13cL76) to make this consistent